examples/randomized-benchmark/two-qubit-rb/example_2qb_rb.py

```diff
@@ -0,0 +1,232 @@
+from rb_2qb import *
+from configuration import config
+from qm.QuantumMachinesManager import SimulationConfig
+from entropylab_qpudb import Resolver
+
+qmm = QuantumMachinesManager()
+
+"""
+For the required two qubit gates to generate the 4 classes (see Supplementary info of this paper:
+https://arxiv.org/pdf/1210.7011.pdf), we require the user to complete the following macros below according
+to their own native set of qubit gates, that is perform the appropriate decomposition and convert the pulse sequence
+in a sequence of baking play statements (amounts to similar structure as QUA, just add the prefix b. before every statement)
+Example :
+in QUA you would have for a CZ operation:
+    play("CZ", "coupler")
+in the macros below you write instead:
+    b.play("CZ", "coupler")
+"""
+# Define here quantum elements required to compute the macros
+q0 = "q0"
+q1 = "q1"
+coupler = "coupler"
+rr1 = "rr1"
+rr2 = "rr2"
+"""
+Define here your Resolver methods such that 
+qubit names "q0" and "q1" are mapped to the correct quantum elements in input config associated 
+to the two qubits you want to benchmark.
+The Resolver allows you to retrieve easily the names of the quantum elements associated to your target qubits, couplers,
+and other elements you might need to carry (fluxlines, etc...).
+
+In this example, we have the simplest case where "q0" and "q1" are defined as the two qubits directly in the config,
+and there is one single coupler named "coupler". Readout resonators coupled to each qubit are named "rr1" and "rr2"
+"""
+
+
+class RBTwoQubitResolver(Resolver):
```

Comment:
the resolver should be specified for a QPU. I.e. if you have say a 10 qubit system with flux tuning and a parameteric coupler, you'd define two channels for the qubit ("xy" and "z"), RR, and a coupler. Then you'd use this mapping in your code.

In this example I'm not entirely sure what the resolver is assisting with

Reply:
Same remark as above, it is not helpful for the specific usecase, but it's put there as a generic expression for this "macro" conception which points out the need to map from circuit level first for Clifford selection before coming back to pulse level